--- origsrc/pymtp-0.0.4/pymtp.py	2008-04-12 12:41:43.000000000 -0500
+++ src/pymtp-0.0.4/pymtp.py	2008-10-13 01:25:22.643067300 -0500
@@ -32,12 +32,18 @@
 __DEBUG__ = 1
 
 import os
+import sys
 import ctypes
 import ctypes.util
 
 # NOTE: This code *may* work on windows, I don't have a win32 system to test
 # this on. 
-_module_path = ctypes.util.find_library("mtp") 
+if sys.platform == 'cygwin':
+    mtplib = "cygmtp-8.dll"
+else:
+    mtplib = "mtp"
+
+_module_path = ctypes.util.find_library(mtplib) 
 _libmtp = ctypes.CDLL(_module_path)
 
 # ----------
